web-api/src/persistence/postgres/utils/migrate/migrations/0007-init-case-correspondence-indexes.ts

```diff
@@ -0,0 +1,16 @@
+import { Kysely } from 'kysely';
+
+export async function up(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .createIndex('idx_caseCorrespondence_docketNumber')
+    .on('dwCaseCorrespondence')
+    .column('docketNumber')
+    .execute();
+}
+
+export async function down(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .dropIndex('idx_caseCorrespondence_docketNumber')
+    .on('dwCaseCorrespondence')
+    .execute();
+}
```

Comment:
Lack of an index was causing a slowdown as well. (It is currently on test--I added through psql.) On our next migration, it would be nice to consolidate some of these migration files.